GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -15,6 +15,8 @@ namespace GVFS.Platform.Mac
     public partial class MacPlatform : POSIXPlatform
     {
         private const string UpgradeProtectedDataDirectory = "/usr/local/vfsforgit_upgrader";
+        private const string DiagnosticReportsDirectory = "/Library/Logs/DiagnosticReports";
+        private const string PanicFileExtention = "*panic";

```

Comment:
Nit: The extension is `"panic"`, this constant might be better described as `PanicFileNamePattern` or perhaps [`PanicFileGlob`](https://en.wikipedia.org/wiki/Glob_(programming)). (I think .NET normally calls them "patterns" so probably best to go with that.)